src/msw/webview_edge.cpp

```diff
@@ -869,6 +750,18 @@ bool wxWebViewFactoryEdge::IsAvailable()
     return wxWebViewEdgeImpl::Initialize();
 }
 
+wxVersionInfo wxWebViewFactoryEdge::GetVersionInfo()
+{
+    IsAvailable(); // Make sure ms_version string is initialized (if available)
+    long versions[3] = { 0, 0, 0 };
+    wxArrayString tokens = wxStringTokenize(wxWebViewEdgeImpl::ms_version, ". ");
+    for (size_t i = 0; i < 3; i++)
```

Comment:
```suggestion
    for (size_t i = 0; i < WXSIZEOF(versions); i++)
```
To be honest I find this entire loop a bit unclear, but I'm not really sure what to replace it with. I'd probably use `wxStringTokenizer` and its `GetNextToken()` in the loop.